deps/libssh2.version

```diff
@@ -0,0 +1,2 @@
+LIBSSH2_BRANCH=master
+LIBSSH2_SHA1=7934c9ce2a029c43e3642a492d3b9e494d1542be
```

Comment:
can we use https://github.com/libssh2/libssh2/releases/tag/libssh2-1.7.0 or is there something on master we need?